tests/flows/basic_erc20_send_receive.go

```diff
@@ -15,46 +15,51 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// BasicERC20SendReceive deploys ERC20Source to subnet A and ERC20Destination to subnet B.
-// Then it sends tokens from subnet A to subnet B and back to subnet A.
 func BasicERC20SendReceive(network interfaces.Network) {
-	subnetAInfo := network.GetPrimaryNetworkInfo()
-	subnetBInfo, _ := teleporterUtils.GetTwoSubnets(network)
+	/**
+	 * Deploy an ERC20 token source on the primary network
+	 * Deploys ERC20Destination to Subnet A
+	 * Bridges C-chain example erc20 tokens to Subnet A
+	 * Bridge tokens from Subnet A to C-chain
+	 */
```

Comment:
Nit: Can you move this comment back above the function definition? It should then show up in certain IDE views when you hover over calls to the function.